webapp/security/views.py

```diff
@@ -220,48 +211,26 @@ def create_notice():
         summary=data["summary"],
         details=data["description"],
         packages=data["releases"],
+        references=data.get("reference", []),
         published=datetime.fromtimestamp(data["timestamp"]),
     )
 
     if "action" in data:
         notice.instructions = data["action"]
 
-    if "isummary" in data:
-        notice.isummary = data["isummary"]
-
     # Link releases
     for release_codename in data["releases"].keys():
-        try:
-            notice.releases.append(
-                db_session.query(Release)
-                .filter(Release.codename == release_codename)
-                .one()
-            )
-        except NoResultFound:
+        release = db_session.query(Release).get(release_codename)
+
+        if not release:
             message = f"No release with codename: {release_codename}."
             return (flask.jsonify({"message": message}), 400)
 
-    # Link CVEs, creating them if they don't exist
```

Comment:
We need to keep this block otherwise no cves or references will be created.

Reply:
But we are already creating CVEs with the API?